test/suites/backup.sh

```diff
@@ -783,3 +783,26 @@ test_backup_volume_rename_delete() {
   lxc storage volume rm "${pool}" vol2
   ! stat "${LXD_DIR}"/backups/custom/"${pool}"/default_vol2 || false
 }
+
+test_backup_different_instance_uuid() {
+  ensure_import_testimage
+  ensure_has_localhost_remote "${LXD_ADDR}"
+
+  echo "==> Checking instances UUID during backup operation"
+  lxc launch testimage c1
+  initialUUID=$(lxc config show c1 --expanded | grep "volatile.uuid:" | awk '{print $2}')
```

Comment:
You can use `lxc config get <instance> <key>` here I think. Should be a bit easier.